dashboards/kv-nodebucket.json

```diff
@@ -196,6 +196,86 @@
       },
       "type": "graph"
     },
+    {
+      "_base": "panel",
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "${node}",
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "unit": "bytes"
+        }
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 10,
+        "w": 9
```

Comment:
Other panels here are all h=10, w=*8* and the 9 here makes the row overflow for me.
![image](https://github.com/couchbaselabs/promtimer/assets/12015256/a3e4b5d1-22d8-4fe8-ba2d-6019f8e59247)